requirements.txt

```diff
@@ -1,6 +1,6 @@
 # regulations-core
 anyjson==0.3.3
-django==1.8.16
+django>=1.8,<1.9
```

Comment:
We can roll out a more comprehensive strategy for this soon. We have some PRs to that effect in progress. This particular upgrade this is needed for ATO and the syntax matches what we are currently doing in the [cms](https://github.com/18F/fec-cms/blob/develop/requirements.txt#L1).